Packages/com.vrchat.ClientSim/Runtime/CameraStacking/ClientSimStackedVRCameraSystem.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.XR;
+using VRC.Core;
+using VRC.SDK3.ClientSim;
+
+namespace VRC.SDK3.ClientSim
+{
+    public class ClientSimStackedVRCameraSystem : MonoBehaviour
+    {
+        public ClientSimStackedCamera[] CameraStack;
+        [HideInInspector] public bool IsCameraStackingEnabled = false;
+        
+        private Camera MainSceneCamera;
+        private bool IsInitialized = false;
+        private bool IsReady = false;
+        private List<Camera> Cameras;
+        
+        private ClientSimMenu clientSimMenu;
+
+        public void Initialize(Camera playerCamera, ClientSimMenu menu)
+        {
+            MainSceneCamera = playerCamera;
+            clientSimMenu = menu;
+        }
+
+        public void Ready()
+        {
+            IsReady = true;
+        }
+        
+        void Update()
+        {
+            if(!IsReady) return;
+            if (!IsInitialized) { InitializeStackedSystem(); }
+            else
+            {
+                if (!Cameras[0].enabled)
+                {
+                    Debug.LogError("Stacked Cameras are not enabled.");
+                }
+            }
+        }
+     
+        /*void OnEnable()
+        {
+            #if !UNITY_ANDROID
+            if (MainSceneCamera != null)
+            {
+                if (!IsCameraStackingEnabled)
+                    CreateCameraStack();
+            }
+            #endif
+        }
+*/
+        void OnDisable()
+        {
+            #if !UNITY_ANDROID
```

Comment:
What happens if this is run with the Android platform target? If this is something that could affect other platforms in the future, we should consider making this an inclusion instead of exclusion (ie `#if UNITY_EDITOR_WIN`)

Reply:
this is an artifact from the client camerastacking, I had assumed it was necessary, I'll go verify the behaviour with xiexe! I'd expect that we use the UNITY_ANDROID pattern a bunch, and that future platform support will expose little things like "this should be checking if windows instead" in a lot of places through our products

Reply:
I've chatted with xiexe and tcl about this, this define is only needed for runtime, not editor, and while it's technically more accurate to disable camera stacking in this way to mirror how our android client works, that's a pretty minor detail, and I've decided that not blocking the clientsim ui (which hinders development) is more important, so I'll clean this up!